pkg/db/db.go

```diff
@@ -315,6 +315,58 @@ type EslEventRow struct {
 	EventJson string
 }
 
+// DBDiscoverCurrentEsldID: Returns the current sequence number of event_sourcing_light table.
+// Next value should be the returned on + 1
+func (h *DBHandler) DBDiscoverCurrentEsldID(ctx context.Context, tx *sql.Tx) (*int, error) {
+	if h == nil {
+		return nil, nil
+	}
+	if tx == nil {
+		return nil, fmt.Errorf("DBDiscoverCurrentEsldID: no transaction provided")
+	}
+	var selectQuery string
+	if h.DriverName == "postgres" {
```

Comment:
It's not great that we have to look at such low level as the sequence number.
For now, we'll merge it, as it seems to work fine, but we may need to reconsider this.